OUX/BasicOUXtvOSPlayer/objc/BasicOUXtvOSPlayer/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder.AppleTV.Storyboard" version="3.0" toolsVersion="11198.2" systemVersion="15G31" targetRuntime="AppleTV" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+    <dependencies>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="11161"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
```

Comment:
@sjbushell you'll have to remove this if you want us to be able to use Xcode 7. Otherwise you'll get an error ""/Users/bsahlas/dev/mobile/ios-root/ios-player-samples/OUX/BasicOUXtvOSPlayer/objc/BasicOUXtvOSPlayer/Base.lproj/Main.storyboard: The document “(null)” requires Xcode 8.0 or later."